conf/iridanext.config

```diff
@@ -13,12 +14,12 @@ iridanext {
                 "**/resfinder.tsv",
                 "**/results.xlsx",
                 "**/settings.txt",
-                "**/summary.tsv"]
+                "**/*summary.tsv"]
```

Comment:
Could you make this pattern more specific? As in `**/staramr/*summary.tsv`?

The pattern is also picking up/treating the `merged_summary.tsv` file as a sample-associated file:

```
            "merged_summary": [
                {
                    "path": "csvtk/merged_summary.tsv"
                }
            ]
```

Reply:
I had changed this, as you suggested, as a fix for the `global` file issue. I didn't commit it because it didn't fix the issue. I used your suggestion `**/*_results/*summary.tsv` which won't pick up the `merged_summary.tsv`